bounded_context/Gemfile.rails_6_0.lock

```diff
@@ -0,0 +1,218 @@
+PATH
```

Comment:
@pawelpacana There was no corresponding lockfile, so I added it.

Reply:
👍 noticed it missing as well, recent addition not caught on CI (#904)

Reply:
👍 noticed it missing as well, recent addition not caught on CI 

> I could not upgrade depencencies in `contrib/ruby_event_store-outbox` due:
> 
> ```
> mutant-dev@mbj ~/devel/rails_event_store/contrib/ruby_event_store-outbox (upgrade/mutant*) $ bundle install --gemfile Gemfile.rails_5_1
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://rubygems.org/..........
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://gem.mutant.dev/.
> Fetching gem metadata from https://rubygems.org/..
> Resolving dependencies...
> Bundler could not find compatible versions for gem "activerecord":
>   In snapshot (Gemfile.rails_5_1.lock):
>     activerecord (= 5.1.7)
> 
>   In Gemfile.rails_5_1:
>     rails (= 5.1.7) was resolved to 5.1.7, which depends on
>       activerecord (= 5.1.7)
> 
>     ruby_event_store-outbox was resolved to 0.0.15, which depends on
>       activerecord (>= 5.2)
> 
> Running `bundle update` will rebuild your snapshot from scratch, using only
> the gems in your Gemfile, which may resolve the conflict.
> ```
> 
> `bundle update` ran into the same dependency violation. The policies on version support, in this project, are unknown to me hence I've to stop at this point.

cc @swistak35

Reply:
found reasoning in bb5257c1464cb253f84825ed66b25d877e2f5e68 and removed extra Gemfiles